Governance.md

```diff
@@ -0,0 +1,78 @@
+# Governance of the rules_scala
+
+These bazel rules for scala are under the bazelbuild org but governed by a
+contributors from the community.
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
```

Comment:
This doesn't render correctly. I think you need a newline here?

Reply:
yep.